setup.cfg

```diff
@@ -37,7 +37,9 @@ show-response = 1
 [tool:pytest]
 minversion = 6.0
 norecursedirs = build docs/_build astroquery/irsa astroquery/nasa_exoplanet_archive astroquery/ned astroquery/ibe astroquery/irsa_dust astroquery/cds astroquery/sha astroquery/dace
-testpaths = astroquery docs
+testpaths =
+    docs/eso
+    astroquery/eso
```

Comment:
remove these changes from the history. You can run just ESO test with pytest out of the box as `pytest -P eso -R` which will pick up only the ESO tests.

(this is also mentioned in the docs, but we may need to add a bit more explanation to that if you think it would have helped you to find the correct command: https://astroquery.readthedocs.io/en/latest/testing.html#running-only-the-remote-data-tests)

Reply:
nono, the docs are totally fine (I have been doing so since the beginning). I'm really sorry to have polluted this PR with this and the changes to the github workflows. My aim was to replicate your CI checks to handle internal Pull Requests. (See https://github.com/juanmcloaiza/astroquery/pulls). We have now a new ESO fork (https://github.com/eso/astroquery/), where such CI checks will live and avoid affecting Pull Requests made to astropy/astroquery. Sorry again for the confusion, I never expected to change the repo configuration.